### PR TITLE
Currency:Addded ubi9.3 build-script,build_info.json for php-src

### PR DESCRIPTION
**Pull request by vinodk99**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Have you validated script on UBI 9 container
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: p/php-src/LICENSE [hunk 1]
```diff
@@ -0,0 +1,68 @@
+--------------------------------------------------------------------
```
Comment (nikhil-kalbande) [inline p/php-src/LICENSE:1]: @vinodk99  we will need apache license in here.. right?
for build script license is always apache

File: p/php-src/php-src_ubi_9.3.sh [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+mv RPM-GPG-KEY-CentOS-Official /etc/pki/rpm-gpg/.
+rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Official
+
+dnf install -y re2c bison autoconf make libtool ccache libxml2-devel sqlite-devel glibc-gconv-extra pcre expect
+
+git clone $PACKAGE_URL
+cd $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+
+./buildconf --force
```
Comment (nikhil-kalbande) [inline p/php-src/php-src_ubi_9.3.sh:42]: is this build kind of command.. if yes lets track exit code as 1 for failure in both buildconf and configure step

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

File: p/php-src/php-src_ubi_9.3.sh (hunk 2 above)
Comment (vinodk99) [reply p/php-src/php-src_ubi_9.3.sh]: @nikhil-kalbande  updated the script as suggested

File: p/php-src/LICENSE (hunk 1 above)
Comment (vinodk99) [reply p/php-src/LICENSE]: @nikhil-kalbande added apache-license

File: p/php-src/php-src_ubi_9.3.sh [hunk 3]
```diff
@@ -47,9 +47,9 @@ if ! ./buildconf --force ; then
 fi
 
 if ! ./configure --enable-debug ; then
-       echo "------------------$PACKAGE_NAME::Configure_Success-------------------------"
+       echo "------------------$PACKAGE_NAME::Configure_fails-------------------------"
        echo "$PACKAGE_URL $PACKAGE_NAME"
-       echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub  | Fail|  Configure_Success"
+       echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub  | Fail|  Configure_fails"
        exit 2
```
Comment (nikhil-kalbande) [inline p/php-src/php-src_ubi_9.3.sh:53]: make exit 1 only here

File: p/php-src/LICENSE [hunk 4]
```diff
@@ -0,0 +1,201 @@
+Apache License
```
Comment (nikhil-kalbande) [inline p/php-src/LICENSE:1]: correct indentation at this line

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

**Review by nikhil-kalbande — APPROVED** [review]

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**